pkg/actions/lua_test.go

```diff
@@ -449,9 +449,10 @@ func TestLuaRunTable(t *testing.T) {
 				Commit: graveler.Commit{
 					Version: 1,
 				},
-				CommitID: "123456789",
-				PreRunID: "3498032432",
-				TagID:    "",
+				CommitID:    "123456789",
+				PreRunID:    "3498032432",
+				TagID:       "tag1",
+				MergeSource: "merge-source",
```

Comment:
Do we have any place in the test where we verify the field? If not - what is the point of adding it?

Reply:
the test uses a golden file which I updated